mitxpro/settings.py

```diff
@@ -295,10 +320,6 @@
     if CLOUDFRONT_DIST:
         AWS_S3_CUSTOM_DOMAIN = "{dist}.cloudfront.net".format(dist=CLOUDFRONT_DIST)
     DEFAULT_FILE_STORAGE = "storages.backends.s3boto.S3BotoStorage"
```

Comment:
Could you swap this to `storages.backends.s3boto3.S3Boto3Storage` so we start this project on the modern `boto3` library since that's been stable for a while?